### DISCUSSION TITLE
Ability to create presets

### DISCUSSION THREAD

**Discussion by gnclmorais** [MonitorControl/MonitorControl#1675] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

I love MonitorControl, but I find myself carefully adjusting the slider a few times every day because I maily oscilate between 85% and 65% of brightness (starting of the day and ending of the day). It’s not a huge problem, but it’s a bothersome to do this every day.

### Describe the solution you'd like

I would like the ability to create a few shortcuts/presets, like a couple of buttons I could press and go straight to custom set percentages — instead of moving the slider trying to get the perfect number.

### Describe alternatives you've considered

None, to be honest. Maybe clicking on the percentage could make it editable, so people could type the target percentage they want, press _Enter_, and it would jump to that?

**Comment by waydabber** [comment, marked as answer]
I don't plan adding presets support to MonitorControl. I recommend using BetterDisplay - it has both Shortcuts support for basic adjustments and a super-extensive CLI support so you can script anything - thus create any kind of preset/script using macOS Shortcuts and a Shell Script action.

https://github.com/waydabber/BetterDisplay/wiki/Integration-features,-CLI

**Reply by gnclmorais** [reply]
Fair enough, thank you for the swift response!
